protocol/app/ante/sigverify_test.go

```diff
@@ -1,26 +1,54 @@
 package ante_test
 
 import (
+	"fmt"
 	"testing"
 
+	storetypes "cosmossdk.io/store/types"
+	"github.com/cosmos/cosmos-sdk/codec"
 	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
 	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
 	"github.com/cosmos/cosmos-sdk/testutil/testdata"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/tx/signing"
 	sdkante "github.com/cosmos/cosmos-sdk/x/auth/ante"
+	authsign "github.com/cosmos/cosmos-sdk/x/auth/signing"
+	authtx "github.com/cosmos/cosmos-sdk/x/auth/tx"
+	txmodule "github.com/cosmos/cosmos-sdk/x/auth/tx/config"
 	"github.com/cosmos/cosmos-sdk/x/auth/types"
-
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	customante "github.com/dydxprotocol/v4-chain/protocol/app/ante"
 	testante "github.com/dydxprotocol/v4-chain/protocol/testutil/ante"
 	clobtypes "github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
 	satypes "github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
-
+	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/require"
 )
 
 func TestSigVerification(t *testing.T) {
 	suite := testante.SetupTestSuite(t, true)
+	suite.TxBankKeeper.EXPECT().DenomMetadata(
+		gomock.Any(),
+		gomock.Any(),
+	).Return(&banktypes.QueryDenomMetadataResponse{}, nil).AnyTimes()
+
+	// signing.SignMode_SIGN_MODE_TEXTUAL and signing.SignMode_SIGN_MODE_LEGACY_AMINO_JSON are disabled
+	// from the test suite since they include the sequence number as part of the signature preventing
+	// us from skipping sequence validation for certain message types.
+	enabledSignModes := []signing.SignMode{signing.SignMode_SIGN_MODE_DIRECT}
```

Comment:
We don't need them and they also don't support skipping sequence numbers since the sequence number is part of the bytes used for signing/signature verification preventing us from practically validating them.